Documentation/interact_v3.md

```diff
@@ -0,0 +1,182 @@
+# Interact with etcd
+
+Users mostly interact with etcd by putting or getting the value of a key. This section describes how to do that by using etcdctl, a command line tool for interacting with etcd server. The concepts described here should apply to the gRPC APIs or client library APIs.
+
+By default, etcdctl talks to etcd server with v2 API for backward compatibility reason. To make etcdctl talks to etcd with v3 API, we first need to set API version to 3 by setting environment. 
+
+``` bash
+export ETCDCTL_API=3
+```
+
+## Write a key
+
+Applications can store keys into etcd cluster. The stored key will be replicated to all etcd members in the cluster through raft to achieve consistency and reliability.
```

Comment:
Applications store keys into the etcd cluster by writing to keys. Every stored key is replicated to all etcd cluster members through raft to achieve consistency and reliability.